### DISCUSSION TITLE
blog/l-alternativa-alle-roadmap/

### DISCUSSION THREAD

**Discussion by giscus** [Cadienvan/Cadienvan.github.io#26] [Announcements]
# blog/l-alternativa-alle-roadmap/

La mia opinione sulle roadmap.

https://cadienvan.github.io/blog/l-alternativa-alle-roadmap/

**Comment by serenasensini** [comment]
Concordo sui Roadmap Token, anche se un intervallo temporale (grande a piacere, in settimane o mesi) è sicuramente utile per dare visibilità a chi sfrutta le nostre soluzioni per poter programmare evolutive o strategie di business. Una roadmap serve anche a tracciare una direzione comune, oltre a rappresentare una scadenza che permette di portare a termine gli obiettivi (non con una certa ansia, a volte necessaria). That's my opinion!

**Reply by Cadienvan** [reply]
Sono d'accordo, infatti dal mio punto di vista la roadmap resta, è il modo con cui si interagisce con essa e le aspettative a riguardo che cambiano!

**Comment by DanieleNebbiai** [comment]
la stai vivendo un po di estremi. 

Roadmap si, roadmap no.

Piazzati nel mezzo con un "Roadmap si, però..", l owner della roadmap quando la presenta deve collegarla a iniziative di business.

Facciamo A perché ci aspettiamo B

E quando qualcuno chiede al team su cosa si sta lavorando, (chiunque nel team compreso prod, tech, design, lead ecc) tutti dovrebbero rispondere "stiamo lavorando su B tramite l' implementazione di A"

Se così non è siamo tutti solo scimmiette 🐒🐒🐒

**Reply by Cadienvan** [reply]
Absolutely, il mio punto è sul prevedere che per arrivare a B non necessariamente si debba fare A, ma anche C. Il problema delle roadmap é che di solito B non lo citano nemmeno e si parla solo di A.
